pynamodb/compat.py

```diff
@@ -77,3 +77,10 @@ def getmembers_issubclass(object, classinfo):
             results.append((key, value))
     results.sort()
     return results
+
+
+# Replace this function with timedelta.total_seconds() once we drop Python 2.6 support
+def total_seconds(delta):
```

Comment:
this returns an integer not a floating point number

Reply:
Indeed, I pulled it from cpython 3.6 without thinking. Good catch.

Reply:
added division future to address this.